core/variant_call.cpp

```diff
@@ -1607,6 +1608,8 @@ void register_variant_methods() {
 	ADDFUNC0R(DICTIONARY, INT, Dictionary, hash, varray());
 	ADDFUNC0R(DICTIONARY, ARRAY, Dictionary, keys, varray());
 	ADDFUNC0R(DICTIONARY, ARRAY, Dictionary, values, varray());
+	//renaming "copy" to "duplicate" to mimic Array and Node method APIs
```

Comment:
How often is `Dictionary.copy()` used through the codebase? Might be worth renaming to `duplicate` to be consistent.

Reply:
> How often is Dictionary.copy() used through the codebase?

If QtCreators reference search is to be trusted, exactly 10 times. ^^

Reply:
Then I'd say go for it.